services/ui_backend_service/data/cache/generate_dag_action.py

```diff
@@ -78,6 +79,7 @@ def execute(cls,
                 keys=None,
                 existing_keys={},
                 stream_output=None,
+                invalidate_cache=False,
```

Comment:
api route accepts, and passes the invalidation boolean to the action. Action does not use it for anything though?

Reply:
This specific action does not have any logic for existing keys(unlike other actions), it will always refetch given location(s). Normally this action will not be executed at all if key exists on disk. This line https://github.com/Netflix/metaflow-service/blob/fix/get-artifact-error-report/services/ui_backend_service/data/cache/client/cache_client.py#L186 is the one that matters in this case.

Reply:
ah I see, this action was using the stream_error and not caching errors to begin with.